Dockerfile

```diff
@@ -526,8 +526,10 @@ ENV MPLBACKEND "agg"
 # Set up an initialization script to be executed before any other commands initiated by the user.
 ADD patches/entrypoint.sh /root/entrypoint.sh
 RUN chmod +x /root/entrypoint.sh
-# This script gets executed by "docker run <image> <command>" and it runs <command> at the end of its execution. 
-ENTRYPOINT ["/root/entrypoint.sh"]
+# This script gets executed by "docker run <image> <command>" and it runs <command> at the end of its execution.
+# NOTE: ENTRYPOINT set by "FROM <image>" should preceed the our own custom entrypoint.
```

Comment:
`preceed` => `precede`

`preceed the our own`.  I think you mean to say something like `The entry point specified by the base image is overriden by our own`.  Perhaps better `This overrides any existing entry point set by the base image`.